tensorflow/core/kernels/data/captured_function.cc

```diff
@@ -448,10 +448,21 @@ Status MakeIteratorFromInputElement(
     const std::vector<Tensor>& input_element, int64 thread_index,
     const InstantiatedCapturedFunction& inst_captured_func, StringPiece prefix,
     std::unique_ptr<IteratorBase>* out_iterator) {
+  return MakeIteratorFromInputElement(
```

Comment:
Here are the internal errors , can you please check 

```
/tensorflow/core/kernels/data/captured_function.cc:450:50: error: all paths through this function will call itself
    std::unique_ptr<IteratorBase>* out_iterator) {
                                                 ^
1 error generated.
/tensorflow/core/kernels/data/captured_function.cc:450:50: error: all paths through this function will call itself 
    std::unique_ptr<IteratorBase>* out_iterator) {
```

Reply:
Michael, you forgot to pass through the `out_iterator` argument.